charts/bulk-scan-cron-trigger/values.yaml

```diff
@@ -4,6 +4,10 @@ job:
   keyVaults:
     bulk-scan:
       secrets:
+        - name: slack-token-daily-checks
+          alias: slack-token-daily-checks
+        - name: slack-channel-id-daily-checks
```

Comment:
Have you added these to the KV in every environment?

Reply:
Yeah dw will do before I merge in haha